#paquete de imagenes RGB y DEPTH

std_msgs/Header header
sensor_msgs/Image imagenRGB
sensor_msgs/Image imagenDEPTH